flink-filesystems/flink-azure-fs-hadoop/pom.xml

```diff
@@ -65,7 +65,6 @@ under the License.
 			<groupId>org.apache.hadoop</groupId>
 			<artifactId>hadoop-hdfs</artifactId>
 			<version>${hadoop.version}</version>
-			<scope>test</scope>
```

Comment:
Why did you remove the `test` scope here?

Reply:
I was running into some issues when I included the shade plugin and tried to run this on the cluster / e2e tests:
```
Caused by: java.lang.NoClassDefFoundError: org/apache/flink/fs/shaded/hadoop3/org/apache/hadoop/hdfs/HdfsConfiguration
	at org.apache.flink.fs.azurefs.AzureFileSystem.createInitializedAzureFS(AzureFileSystem.java:48)
```

Reply:
Thanks for the clarification.